UI/src/controller/speedLimitsCollection.js

```diff
@@ -179,7 +186,7 @@
     };
 
     this.saveSplit = function(callback) {
-      backend.splitSpeedLimit(splitSpeedLimits.existing.id, splitSpeedLimits.splitMeasure, splitSpeedLimits.created.value, splitSpeedLimits.existing.value, function() {
+      backend.splitSpeedLimit(splitSpeedLimits.existing.id, splitSpeedLimits.splitMeasure, splitSpeedLimits.created.value.value, splitSpeedLimits.existing.value.value, function() {
```

Comment:
Can this "value.value" be changed to something more explicit.

Reply:
Let's leave that "value.value" like it is because to switch that we will have a lot of impact on the rest of the code.